.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -667,6 +670,12 @@ tsh ssh node-that-requires-device-trust
   - [ ] Non-signed and/or non-notarized `tsh` for macOS gives a sane error
         message for `tsh device enroll` attempts.
 
+- [ ] Device support commands
+  - [ ] `tsh device collect`   (macOS)
+  - [ ] `tsh device asset-tag` (macOS)
```

Comment:
Not related to this PR, but looks like we can deprecate `tsh device collect` in favor of `tsh device asset-tag`?

Reply:
`tsh device collect` is useful for debugging, I'd rather keep it.